app/models/manageiq/providers/vmware/infra_manager/refresher.rb

```diff
@@ -289,9 +271,11 @@ def reconfig_refresh
         dummy, timings = Benchmark.realtime_block(:total_time) do
           Benchmark.realtime_block(:get_vc_data_total) do
             begin
-              get_vc_data_by_mor(ems, :vm, vm.ems_ref_obj)
-              get_vc_data_by_mor(ems, :host, vm.host.ems_ref_obj)
-              get_vc_data_by_mor(ems, :storage, vm.host.storages.collect(&:ems_ref_obj))
+              accessors = [:vm, :host, :storage].each_with_object([]) do |type, m|
+                m.push([VC_ACCESSORS_BY_MOR[type], type])
+              end
```

Comment:
This looks like a lot of work, might be better to change `VC_ACCESSORS` to be a hash and get rid of `VC_ACCESSORS_BY_MOR`?